.github/workflows/workflow-release-services.yaml

```
@@ -63,6 +65,14 @@ jobs:
       - name: Build and push
         uses: docker/build-push-action@v5
         with:
-          context: ./digiwf-deployment-service
+          context: ./digiwf-cocreation-deployment-service
           push: true
-          tags: itatm/digiwf-deployment-service:${{ inputs.release-version }},itatm/digiwf-deployment-service:dev
+          tags: itatm/digiwf-cocreation-deployment-service:${{ inputs.release-version }},itatm/digiwf-cocreation-deployment-service:dev
+
+      - name: Update digiwf-cocreation-deployment repo description
```

```suggestion
      - name: Update digiwf-cocreation-deployment repo description
        if: inputs.snapshot-release == false
```
Würde das wahrscheinlich auch nur bei einem normalen Release machen oder was meinst du?